graylog2-web-interface/src/components/sidecars/sidecars/VerboseMessageModal.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import lodash from 'lodash';
+
+import BootstrapModalWrapper from 'components/bootstrap/BootstrapModalWrapper';
+import CombinedProvider from 'injection/CombinedProvider';
+
+const { CollectorConfigurationsActions } = CombinedProvider.get('CollectorConfigurations');
+
+class VerboseMessageModal extends React.Component {
+
```

Comment:
There should be no white line here, as it breaks the padded-blocks rule: https://eslint.org/docs/rules/padded-blocks

```suggestion
```